Blog/sammoss.md

```diff
@@ -0,0 +1,8 @@
+I know Corbett's article is really a sidenote to this, but I just want to add, they also resonated with me because I actually worked ETDs in the Graduate Shool of my alma mater, Univ of North TX, one of the big Tier 1 institutions in TX. The libraries are absolutely outstanding... but they did nothing with ETDs except file them at the time (we were working on more collab). So as the part-time assistant to the severely overworked Grad Reader for the NTX System, I was constantly fielding concerns and questions about Proquest (always looking to make a buck), copyright (which Proquest paperwork tries to make you think you need to pay to be copyrighted). That's all. Hopefully the phenomenal library system has been able to take more of a role so my counterpart is not having bad ETD dreams. 
+McGrath's post made me put 2+2 together that Digi Hum is not just coding. I mean, I knew that?? But maybe coding is such a forward face to the 'discipline'. Discussion of polyvocality also touch on some of the questions I'm thinking about in my own work right now in Cultural Heritage and Preservation Studies (CHAPS). I've been really thinking about the tensions between expert voices and local voices, a debate which is central to CHAPS concerns. I can't even begin to get into it here, but that does bring me to Shroeders's article. I realize last week that I stepped into a bit when I was asked to introduce myself and my thoughts on Digi Hum. I mentioned the tension bewteen the ideas of "fieldification" and inter/trans-disciplinarity. I know I was directly translating from Public Humanities, but I think it's a legit question to ask of Digi Hum. And buried within that tension of "fieldification" and the inter/trans-disciplinarity is ultimately a deep tension regarding expertise. That's just my quick gloss on it. And this really leads me to think about Risam's article again. There's always a question that I think should be in the back of mind, and that is who is allowed or not allowed to participate, collaborate, share their voice, etc.? Something can look polyvocal, but is it? Are there barriers? Just thoughts. 
```

Comment:
The "fieldification" in DH is something I thought about in light of Schroeders's article, too. And how can Public Humanities and DH intersect / augment / or remain outside of more "traditional" boundaries." And we did not get to talk about McGrath, but podcast as both a site of oral history archives and living history certainly fits inside of both DH and Public Humanities concerns.